flexinput/src/main/java/com/lytefast/flexinput/adapters/PhotoCursorAdapter.kt

```diff
@@ -136,12 +165,25 @@ class PhotoCursorAdapter(private val contentResolver: ContentResolver,
     fun bind(photo: Photo?) {
       this.photo = photo
 
-      val thumbnailUri = photo?.let {
-        setSelected(selectionCoordinator.isSelected(photo, adapterPosition), false)
-        it.getThumbnailUri(contentResolver)
-      }
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+        imageView.hierarchy.setPlaceholderImage(placeholderDrawable, ScalingUtils.ScaleType.CENTER)
+
+        loadingThumbnailJob = GlobalScope.launch(context = Dispatchers.Main) {
```

Comment:
can you explain what the context does here? do we need to specify a context?

Reply:
Yea will do